File: Java/org/mayope/api/response/SuccessResponse.java
package org.mayope.api.response;

/**
 * A geneic boolean success/failure response.
 */
public class SuccessResponse extends Response {
    /**
     * True if the operation succeeded, false otherwise.
     */
    private boolean success;

    /**
     * Get true if the operation succeeded, false otherwise.
     *
     * @return The value
     */
    public boolean getSuccess() {
        return success;
    }

    /**
     * Set true if the operation succeeded, false otherwise.
     *
     * @param success The new value
     */
    public void setSuccess(boolean success) {
        this.success = success;
    }
}

File: CSharp/Mayope.Api/Responses/SuccessResponse.cs
using System.ComponentModel.DataAnnotations;

using Newtonsoft.Json;

namespace Mayope.Api.Responses;
{
    /// <summary>
    /// A geneic boolean success/failure response.
    /// </summary>
    public abstract class SuccessResponse : IResponse
    {
        /// <summary>
        /// True if the operation succeeded, false otherwise.
        /// </summary>
        [Required]
        [JsonRequired]
        public bool Success { get; set; }
    }
}
